src/value/mod.rs

```diff
@@ -838,8 +838,11 @@ mod generic_value_tests {
         }
     }
 
+    // A struct that defines input case for `struct_accessors` method
```

Comment:
Minor, especially since this will not get docs generated, but I like the consistency:

```suggestion
    /// A struct that defines input case for `struct_accessors` method
```